src/os/bluestore/BlueFS.cc

```diff
@@ -606,6 +613,140 @@ void BlueFS::_stop_alloc()
   }
 }
 
+int BlueFS::read(uint8_t ndev, uint64_t off, uint64_t len,
```

Comment:
It might be a nice to log func name and func params at e.g. level 10 to be able to track if we're getting into this method.
The same applies to read_random....

Reply:
Done.